website/docs/interactors/2-built-in-dom.md

```diff
@@ -49,6 +52,14 @@ export default test('BigTest Runner')
   .assertion(Page.has({ title: 'BigTest Example App'}));
 ```
 
+### HTML
+
+The `HTML` interactor is not meant to be used directly in your tests but for _composing_ other interactors. How this interactor is used will make much more sense once you read the `Write Your Own Matchers` page. But for now you can think of the `HTML` interactor as the basic building block for all other interactors.
+
+We took many of the common HTML properties and interactions, such as `className` and `click`, and added them into the `HTML` interactor. This provides the convenience of not having to respecify these properties over and over again for each of your interactors.
+
+Take a look at the [source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts) for the `HTML` interactor to see which filters and actions were added.
```

Comment:
This one was tricky to discuss. I think a quick description here is warranted because it's unique from the other built-in interactors just like `Page`. ("Why is HTML there? Is Button and CheckBox _not_ HTML?")

But at this point in the docs, the reader hasn't been exposed to the interactor properties yet so they won't know what `filters` or `locators` are nor will they understand what `"extending" interactors` means so I felt like I needed to be as generic and vague as possible.

Reply:
That's true. HTML doesn't really fit here because `built-in` sort of implies it's something you can use "out-of-the-box" in your tests. Thoughts @jenweber?

Reply:
@jenweber Also, in response to your comment, I agree that option 2 is the right way to go at the moment (without API docs). I skimmed through the docs and I think if we...
- Scrub all the instances of us saying "take a look at the source code" on `Built-In Interactors` and `Locators, Filters, and Actions` page
- Remove the `HTML` section on the `Built-In Interactors` page

... then we could add a section _before_ the construction of the custom interactor. So it would be like "Here are the parts that make up an interactor: 1, 2, 3, 4. Putting this together ... this is how you do it."

And before we show the actual example, we could say "But first, we need to explain how extending works."

At the moment I have the long explanation after the fact but I think that'll raise questions when they encounter it without knowing what it is so it would be better to explain it all before we show the example.